SU-SOE/stanford_soe_helper
# 7.x 2.x featured

# READY FOR REVIEW

# Summary
- This updates the stanford_event_featured view to show the most recent event if there's no featured event. 
- This uses an alter hook to add  VBO, exposed filter, and featured fields to the manage content view.

# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
For homepage block:
1) Verify event block on homepage is displaying a featured event (you can check manage content at /admin/manage/events)
2) Remove "featured" from all events
3) Verify the event block on the homepage displays the most recent event.

Repeat for homepage news block


For manage content
1) Navigate to /admin/manage/events
2) Verify that you see  an exposed filter for featured and a **featured** column in the table
3) Select several events and verify you can use VBO to change their status from **Not Featured** to **Featured** and visa versa.

Repeat for news at /admin/manage/stanford_news_item

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
- https://stanfordits.atlassian.net/browse/SOE-2170
- https://stanfordits.atlassian.net/browse/SOE-2178
## Related PRs
https://github.com/SU-SWS/stanford_magazine/pull/20
## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
